custom_controller:
  ros__parameters:
    # Controller type
    type: ros2_controller_no_pub_sub_pkg/CustomController

    # Controller-specific parameters
    k_gains:
      - 24.0
      - 24.0
      - 24.0
      - 24.0
      - 10.0
      - 6.0
      - 2.0
    d_gains:
      - 2.0
      - 2.0
      - 2.0
      - 1.0
      - 1.0
      - 1.0
      - 0.5
    